content/practices/high-contrast/high-contrast-practice.html

```diff
@@ -0,0 +1,779 @@
+        <table class="data">
+          <caption>Operating System Contrast Adjustment Features</caption>
+          <thead>
+            <tr>
+              <th>Feature</th>
+              <th abbr="Operating Systems">OS</th>
+              <th>Description</th>
+              <th>Authoring Practices</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <th>Invert Colors</th>
+              <td>
+                <ul>
+                  <li><a href="https://support.apple.com/en-ge/guide/mac-help/mchl11ddd4b3/mac">macOS</a></li>
+                  <li><a href="https://support.microsoft.com/en-us/office/use-color-and-contrast-for-accessibility-in-microsoft-365-bb11486d-fc7d-4cd9-b344-16e2bc2a2387#bkmk_invertwin11">Windows</a></li>
+                  <li><a href="https://support.apple.com/en-ge/guide/iphone/iph3e2e1fb0/ios">iOS</a></li>
+                  <li><a href="https://support.google.com/accessibility/android/answer/6151800?hl=en">Android</a></li>
+                  <li><a href="https://help.gnome.org/users/gnome-help/stable/a11y-mag.html.en">Linux/GNOME</a></li>
+                </ul>
+              </td>
+              <td>
+                <p>
+                  Invert colors transforms colors by subtracting each RGB value from 255.
+                  This is a simple way of turning a light color theme into a dark color theme or vice-versa.
+                  It is a prominent feature in the accessibility settings of most operating systems.
+                </p>
+                <p>Examples of color inversion:</p>
+                <ul>
+                  <li>rgb(255, 255, 255) is rendered as rgb(0, 0, 0)</li>
+                  <li>rgb(165, 42, 42) is rendered as rgb(90, 213, 213)</li>
+                  <li>rgb(220, 220, 220) is rendered as rgb(35, 35, 35)</li>
+                </ul>
+              </td>
+              <td>
+                Browsers do not provide a way for authors to determine whether a user has the setting for invert colors enabled.
+                For invert colors to work well, ensure content meets the minimum color requirements specified by
+                <a href="https://www.w3.org/TR/WCAG22/">Web Content Accessibility Guidelines</a> (WCAG).
+              </td>
+            </tr>
+            <tr>
+              <th>Increase contrast</th>
+              <td>
+                <ul>
+                  <li><a href="https://support.apple.com/guide/mac-help/change-display-settings-for-accessibility-unac089/mac">macOS</a></li>
+                  <li><a href="https://support.microsoft.com/en-us/windows/change-color-contrast-in-windows-fedc744c-90ac-69df-aed5-c8a90125e696">Windows</a></li>
+                  <li><a href="https://support.apple.com/en-ge/guide/iphone/ipheed8d2fca/18.0/ios/18.0">iOS</a></li>
+                  <li><a href="https%3A//support.google.com/accessibility/android/answer/11183305%3Fhl%3Den%23zippy%3D%252Cuse-high-contrast-text">Android</a></li>
```

Comment:
```suggestion
                  <li><a href="https://support.google.com/accessibility/android/answer/11183305">Android</a></li>
```

@jongund the link is certainly valid and something to look more into about why this structure is invalid, but the suggestion gets to the same result, so it may be better to keep it trimmed.

Navigating to another link on that page and then navigating back yields additional session information in the URL for myself and I'm thinking that may be where `#zippy=` is coming from? (some special session value when this link was copied and pasted). Additionally, the `?hl=en` will be forcing the user to visit the page with it auto-translated to English which may not be what all users want.